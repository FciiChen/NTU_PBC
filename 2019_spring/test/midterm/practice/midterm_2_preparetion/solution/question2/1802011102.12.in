2rDuLF, 6jrU,IetiYZp,tqf3T,ZwI,MrMO,xkboHkA
WJsbvHnf6, SLsptA0hS,p59,owV9Ctd,ubyz,8p0rC4xWGh,bI5cWuBPL
1Ft, xC9H8E684,d2o,SiOn,fEUgTz,EOPGeIB,3kUXA
Ce5, X3j,Jqvs0m,4DgCIuqHM,UusJEV3q,ftkS6A3t,0dmkIdT
l91y8Pg5N, ZEtvJgJV,vQyI9,jKpdRdHB8u,lP9T,prO,Q9Jb
8gtRj, nj0SdLIt3,risurXC,eq1MymbKEd,kMGcv7N2,mJ4,rsmvu
hXKXlCC6g, LqihYSwut,t6wmwIwo,BLt48gl6lU,L5SCG5qoH,YhiQ,fWj16
BREAK
2rDuLF ,  AjFVOG3p9
BREAK
