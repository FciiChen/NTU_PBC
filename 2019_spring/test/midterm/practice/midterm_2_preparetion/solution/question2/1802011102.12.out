2rDuLF     6jrU       IetiYZp    tqf3T      ZwI        MrMO       xkboHkA    AjFVOG3p9 
WJsbvHnf6  SLsptA0hS  p59        owV9Ctd    ubyz       8p0rC4xWGh bI5cWuBPL  NULL      
1Ft        xC9H8E684  d2o        SiOn       fEUgTz     EOPGeIB    3kUXA      NULL      
Ce5        X3j        Jqvs0m     4DgCIuqHM  UusJEV3q   ftkS6A3t   0dmkIdT    NULL      
l91y8Pg5N  ZEtvJgJV   vQyI9      jKpdRdHB8u lP9T       prO        Q9Jb       NULL      
8gtRj      nj0SdLIt3  risurXC    eq1MymbKEd kMGcv7N2   mJ4        rsmvu      NULL      
hXKXlCC6g  LqihYSwut  t6wmwIwo   BLt48gl6lU L5SCG5qoH  YhiQ       fWj16      NULL      
